nginx-mainline.yaml

```diff
@@ -338,9 +338,9 @@ test:
 
 update:
   enabled: true
-  github:
-    identifier: nginx/nginx
-    use-tag: true
-    # MAINLINE VERSIONS MUST USE ODD MINOR VERSIONS (e.g., 1.27.x, 1.29.x)
-    tag-filter: release-1.29
+  # MAINLINE VERSIONS MUST USE ODD MINOR VERSIONS (e.g., 1.27.x, 1.29.x)
```

Comment:
```suggestion
  # MAINLINE VERSIONS MUST USE ODD MINOR VERSIONS (e.g., 1.27.x, 1.29.x)
  # SO IGNORE ANY TAG THAT IS NOT ODD
```

Reply:
I know the next line uses `ignore-regex-patterns` but my eye/brain skipped over that and started wondering why the regex contains even numbers when the comment mentions odd ones.